### ISSUE TITLE
Nice overview of Git!

### ISSUE THREAD

**Issue by alancohen** [stephensxu/phase_0_unit_1#5]
Nice overview of Git!  Question - can you do a git commit without using the '-m [message]'?  I'm so used to using it now, but I never thought about if it is actually required or not.

**Comment by stephensxu** [comment]
Yes you can commit without using the -m. If you do "git commit"; it will direct you to your default git text editor and ask you to enter your commit message there. Git commit -m [message] is some sort of short-cut.
